cfme/tests/infrastructure/test_advanced_search_host.py

```diff
@@ -75,7 +75,7 @@ def test_host_filter_without_user_input(host_collection, hosts, hosts_with_vm_co
     hosts_advanced_search.flash.assert_no_error()
 
     view = host_collection.appliance.browser.create_view(HostsView)
-    hosts_on_page = len(view.entities.get_all())
+    hosts_on_page = len(view.entities.entity_names)
```

Comment:
Question:
what is a problem with `get_all` method?

Reply:
@digitronik 
Yes, there is no difference between **get_all()** method and **entity_names**.
I tried test with both methods locally but did not found error this time.
Hence closing this PR.

Thank you.